roles/cifmw_cephadm/tasks/dashboard/validation.yml

```diff
@@ -1,15 +1,47 @@
+---
+# Copyright 2024 Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Update urischeme based on cert/key
+  ansible.builtin.set_fact:
+    cifmw_cephadm_urischeme_dashboard: "https"
+  when:
+    - cifmw_cephadm_dashboard_crt | length > 0
```

Comment:
The job is currently failing because `cifmw_cephadm_dashboard_crt' is undefined`.
I suppose we want to put an additional `cifmw_cephadm_dashboard_crt is defined` in the condition